### DISCUSSION TITLE
Problems installing the new PS1 script with the new version of Remotely

### DISCUSSION THREAD

**Discussion by alencarwell** [immense/Remotely#851] [General]
Hello friends, how are you.
Could you help me with a question?

I'm using the latest version of Remotely and now we see that only Windows 10 and Windows 11 are compatible with the solution.

As there are Builds of the most varied versions and each operating system has its updates installed differently, that is, not all updates are present in the operating systems.

I'm noticing enormous difficulty in running the ps1 script on certain machines, that is, on some it works and others it doesn't, I don't know if it's due to differences in Builds or KB on each machine.

My park has a few hundred machines and getting them all up to standard is very difficult.

Is there a way to make this ps1 script not make these specific KBs and/or Build requirements? I don't even know if this is the problem, but from the tests I'm doing, it seems that this script does not apply to all operating systems, not in this case, Windows 10 and 11.

To avoid this problem, it would be good if the agent was an MSI installer, because applying the ps1 script GPO is very restrictive by Microsoft itself, whereas MSI installers are the default applied by the GPO, understand.

I am very grateful for your help.

**Comment by alencarwell** [comment]
PS C:\Windows\system32> Set-ExecutionPolicy Bypass

PS C:\Windows\system32> C:\temp\Install-Remotely-IP.ps1
Install/uninstall logs are being written to "C:\Users\ADM~1.SIL\AppData\Local\Temp\Remotely_Install.txt"
Install started.
Error occurred: At C:\temp\Install-Remotely-IP.ps1:107 char:18
+ ... dResponse = Invoke-WebRequest -Uri "$HostName/Content/Remotely-Win-$P ...
Exiting...

PS C:\Windows\system32>

**Comment by alencarwell** [comment]
OS Windows 10 Enterprise 21H1 Build 19043.1052

**Comment by CMcNichol-GBSTP** [comment]
Hi @alencarwell 

I currently install Remotely on my PCs with a GPO running the PowerShell script without issues.
Did you check the log file, as that may point you further in the right direction rather than just the error that PowerShell gives?


Chris
